### PR TITLE
fix: mobile view: need adjustment in elements

### LINKED ISSUE #6146: Need Adjustment in elements while seeing in mobile view
Screenshot (111)
**Describe the bug**

There should be a gap between these three buttons or need proper alignment

**Screenshots**

### PR DESCRIPTION
**Pull request by afzal442**
Fixes #6146 

#### Changes proposed in this pull request:

- Improved alignments
- Improved spaces
-

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by afzal442** [comment]
Updated other things as well
Before
c3

After
c5

**Comment by MeghalBisht** [comment]
#6152 is already opened, why a new PR?

**Comment by afzal442** [comment]
>why a new PR?

I knew but previous one was not upto the mark then I suggested that fellow to update. Then again I waited for his comment for some time so I created another PR which is resolving more than the requirements. Thanks. Even if there is more than one PR created, it will matter as per @mariobehling suggestions. U might have observed on others issues if not I can give u reference. Thanks.

**Comment by iamareebjamal** [comment]
resolve conflicts

**Comment by afzal442** [comment]
@iamareebjamal sir you might have observed I suggested that fellow and later he copied my code. Thanks.

**Comment by iamareebjamal** [comment]
You should have waited for him to update the PR

**Comment by himanshujaidka** [comment]
> @iamareebjamal sir you might have observed I suggested that fellow and later he copied my code. Thanks.

I think that u copied the file location after my first pr

**Comment by himanshujaidka** [comment]
> > @iamareebjamal sir you might have observed I suggested that fellow and later he copied my code. Thanks.
> 
> I think that u copied the file location after my first pr

or u should make a pr before I created

**Comment by himanshujaidka** [comment]
@afzal442 the difficult task is to find the file location

**Comment by afzal442** [comment]
> You should have waited for him to update the PR
> 
> You suggested mt class, he used mt class. That's not copying

No sir u can see my PR updated time as well. He referred my code as soon as I created.

**Comment by afzal442** [comment]
>the difficult task is to find the file location

I already explored before but unable to reach the solution. BTW I have collaborated with others as well but didn't find such unexpected things in this community.

**Comment by iamareebjamal** [comment]
You should have waited for him to update the PR

You suggested mt class, he used mt class. That's not copying

Why did you suggest if you didn't want him to copy?

Why did you not wait after suggesting for him to update the PR?

**Comment by himanshujaidka** [comment]
> > the difficult task is to find the file location
> 
> I already explored before but unable to reach the solution. BTW I have collaborated with others as well but didn't find such unexpected things in this community.

I think u don't know about the basic FIFO system

**Comment by afzal442** [comment]
> You should have waited for him to update the PR
> 
> You suggested mt class, he used mt class. That's not copying

Sorry sir, but I waited more than half an hour after suggesting. It is quite obvious that he is naive here, he is trying to explore every file and even he didn't know the use of mt class before. So he referred my code. Thanks.

**Comment by iamareebjamal** [comment]
Half an hour is nothing. Wait for a day and ask

**Comment by afzal442** [comment]
> Half an hour is nothing. Wait for a day and ask
> 

Sorry, Sir, I will take care. Thanks

**Comment by afzal442** [comment]
> You were once as well

Yep, I knew but didn't copy someone's PR code as soon as created rather I told something else to ask for collaboration. You might have observed. [see comment here](https://github.com/fossasia/open-event-frontend/pull/6096#issuecomment-749601222) Thanks.

**Comment by himanshujaidka** [comment]
> > You were once as well
> 
> Yep, I knew but didn't copy someone's PR code as soon as created rather I told something else to ask for collaboration. You might have observed. [see comment here](https://github.com/fossasia/open-event-frontend/pull/6096#issuecomment-749601222) Thanks.

It is not a big issue that we have to create some big logic for this. You are behaving like that you created or discovered a new algo and someone copied it. If you have a problem then please attach copyright after every PR from NOW ONWARDS

**Comment by himanshujaidka** [comment]
> > > You were once as well
> > 
> > 
> > Yep, I knew but didn't copy someone's PR code as soon as created rather I told something else to ask for collaboration. You might have observed. [see comment here](https://github.com/fossasia/open-event-frontend/pull/6096#issuecomment-749601222) Thanks.
> 
> It is not a big issue that we have to create some big logic for this. You are behaving like that you created or discovered a new algo and someone copied it. If you have a problem then please attach copyright after every PR from NOW ONWARDS

and If you are so excited to solve bugs then go and solve big issues don't look for easy issues if you are not at the early stage of contribution. THANKS

**Comment by afzal442** [comment]
@himanshujaidka congrats for your first PR here in this project. BTW plz explore other resolved issues and PR's and then come up with a solution. I knew you are stronger than me. Sorry if I hurted you. I think I need to understand collaboration which I did a collaboration with u as well. Thanks.

**Comment by himanshujaidka** [comment]
> @himanshujaidka congrats for your first PR here in this project. BTW plz explore other resolved issues and PR's and then come up with a solution. I knew you are stronger than me. Sorry if I hurted you. I think I need to understand collaboration which I did a collaboration with u as well. Thanks.

The thing That I don't want to say or it will not suite but for you I am telling that this issue was discovered by me and on that I clearly mention that look for other issue don't try to take that I am already working, I appreciate your solution that u have suggested me of mt class but don't play a blame game, be mature

File: app/components/forms/wizard/wizard-footer.hbs [hunk 1]
```diff
@@ -33,7 +33,7 @@
   </div>
 </div>
 <div class="ui grid">
-  <div class="sixteen wide column right aligned">
+  <div class="sixteen wide column left aligned">
```
Comment (iamareebjamal) [inline app/components/forms/wizard/wizard-footer.hbs:36]: Why?

Comment (afzal442) [reply app/components/forms/wizard/wizard-footer.hbs:36]: Plz, [refer](https://github.com/fossasia/open-event-frontend/pull/6153#issuecomment-751460775) to this above. Thanks Sir.

Comment (maze-runnar) [inline app/components/forms/wizard/wizard-footer.hbs:36]: it shouldn't be left aligned for desktop view.

Comment (afzal442) [reply app/components/forms/wizard/wizard-footer.hbs:36]: Thanks, should I use if condition for mobile view only?

**Comment by afzal442** [comment]
Before
c5
After
c4

File: app/components/forms/wizard/wizard-footer.hbs [hunk 2]
```diff
@@ -33,7 +33,7 @@
   </div>
 </div>
 <div class="ui grid">
-  <div class="sixteen wide column right aligned">
+  <div class="sixteen wide column {{unless this.device.isMobile 'left aligned'}}">
```
Comment (maze-runnar) [inline app/components/forms/wizard/wizard-footer.hbs:36]: unless mobile -> right aligned

Comment (afzal442) [reply app/components/forms/wizard/wizard-footer.hbs:36]: but this will revert the changes. mobile view will be changed to before one as above.

Comment (maze-runnar) [reply app/components/forms/wizard/wizard-footer.hbs:36]: I think you are confused between if and unless

Comment (afzal442) [reply app/components/forms/wizard/wizard-footer.hbs:36]: No I already checked it. But it's showing right aligned despite using unless.

Comment (sachinchauhan2889) [inline app/components/forms/wizard/wizard-footer.hbs:36]:
```suggestion
  <div class="sixteen wide column {{if this.device.isMobile 'left aligned' 'right aligned'}}">
```

**Review by sachinchauhan2889 — CHANGES_REQUESTED** [review]
please commit my suggested changes. We need to merge it as soon as possible.

**Review by sachinchauhan2889 — CHANGES_REQUESTED** [review]
not working

File: app/components/forms/wizard/wizard-footer.hbs [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <div class="ui grid">
   <div class="sixteen wide column {{unless this.device.isMobile 'right aligned'}}">
```
Comment (sachinchauhan2889) [inline app/components/forms/wizard/wizard-footer.hbs:2]:
```suggestion
  <div class="sixteen wide column {{unless @device.isMobile 'right aligned'}}">
```

File: app/components/forms/wizard/wizard-footer.hbs [hunk 4]
```diff
@@ -33,7 +33,7 @@
   </div>
 </div>
 <div class="ui grid">
-  <div class="sixteen wide column right aligned">
+  <div class="sixteen wide column {{if this.device.isMobile 'left aligned' 'right aligned'}}">
```
Comment (sachinchauhan2889) [inline app/components/forms/wizard/wizard-footer.hbs:36]:
```suggestion
  <div class="sixteen wide column {{unless @device.isMobile 'right aligned'}}">
```

**Review by sachinchauhan2889 — CHANGES_REQUESTED** [review]
wherever you see this in codebase
## <Forms::Wizard::WizardFooter/>

pass @device={{this.device}}

**Review by sachinchauhan2889 — CHANGES_REQUESTED** [review]
this will not work. 
you are not getting my point. 
here in this file @device is undefined. 
you need to pass them from wherever we are using this component. 
@device={{this.device}}

File: app/templates/components/forms/wizard/basic-details-step.hbs [hunk 5]
```diff
@@ -616,6 +616,7 @@
   @event={{this.data.event}}
   @loading={{this.loading}}
   @first={{true}}
+  @device={{this.device}}
```
Comment (sachinchauhan2889) [inline app/templates/components/forms/wizard/basic-details-step.hbs:619]: what about other places.
there are 4 places where we are using this object.
update on other three places

**Review by sachinchauhan2889 — CHANGES_REQUESTED** [review]

File: app/templates/components/forms/wizard/basic-details-step.hbs (hunk 5 above)
Comment (afzal442) [reply app/templates/components/forms/wizard/basic-details-step.hbs:619]: done.

**Review by sachinchauhan2889 — APPROVED** [review]

**Pull request merged**